google/cloud/bigtable/row_response.py

```diff
@@ -114,17 +342,91 @@ def __int__(self) -> int:
         """
         return int.from_bytes(self.value, byteorder="big", signed=True)
 
+    def to_dict(self) -> dict[str, Any]:
+        """
+        Returns a dictionary representation of the cell in the Bigtable Cell
+        proto format
+
+        https://cloud.google.com/bigtable/docs/reference/data/rpc/google.bigtable.v2#cell
+        """
+        cell_dict: dict[str, Any] = {
+            "value": self.value,
+        }
+        cell_dict["timestamp_micros"] = self.timestamp_micros
+        if self.labels:
+            cell_dict["labels"] = self.labels
+        return cell_dict
+
     def __str__(self) -> str:
         """
         Allows casting cell to str
         Prints encoded byte string, same as printing value directly.
         """
         return str(self.value)
 
+    def __repr__(self):
+        """
+        Returns a string representation of the cell
+        """
+        return f"CellResponse(value={self.value!r}, row={self.row_key!r}, family='{self.family}', column_qualifier={self.column_qualifier!r}, timestamp_micros={self.timestamp_micros}, labels={self.labels})"
+
     """For Bigtable native ordering"""
 
     def __lt__(self, other) -> bool:
-        raise NotImplementedError
+        """
+        Implements `<` operator
+        """
+        if not isinstance(other, CellResponse):
+            return NotImplemented
+        this_ordering = (
+            self.family,
+            self.column_qualifier,
+            -self.timestamp_micros,
```

Comment:
please dont fight native ordering

Reply:
My understanding was that newer cells should come first? By default, older timestamps would come first.

That said, comparison is less relevant if we just use cells as they come from the backend though. If we don't have a way to determine proper ordering from the client side, maybe we should remove these comparison implementations entirely?